contracts/core/interfaces/IMinterModule.sol

```diff
@@ -87,7 +87,7 @@ interface IMinterModule is IERC165 {
     event Minted(
         address indexed edition,
         uint128 indexed mintId,
-        address indexed buyer,
+        address indexed to,
```

Comment:
is this a breaking change for indexers?

Reply:
Nope. The signature only depends on the type of the args.

See: https://github.com/chiru-labs/ERC721A/blob/1843596cf863557fcd3bf0105222a7c29690af5c/contracts/ERC721A.sol#L89